### PR TITLE
Remove CERN themes from the core

### PR DESCRIPTION
**Pull request by ThiefMaster**
Added as a CERN plugin in https://gitlab.cern.ch/indico/indico-plugins-cern/merge_requests/11

### CODE REVIEW

File: indico/modules/events/timetable/views/__init__.py [hunk 1]
```diff
@@ -132,10 +132,14 @@ def inject_meeting_body(event, **kwargs):
     days = [(day, list(e)) for day, e in groupby(entries, lambda e: e.start_dt.astimezone(event_tz).date())]
     theme_id = view if view and view in theme_settings.themes else event.theme
     theme = theme_settings.themes[theme_id]
-    tt_tpl = theme.get('tt_template', theme['template'])
+    plugin = theme.get('plugin')
+    tpl_name = theme.get('tt_template', theme['template'])
+    tt_tpl = (plugin.name + tpl_name
+              if plugin and tpl_name[0] == ':'
```
Comment (pferreir) [inline indico/modules/events/timetable/views/__init__.py]: Maybe this is one of those cases where parenthesis would help to understand the precedence of the `+` operator over the `if`.

Comment (ThiefMaster) [inline indico/modules/events/timetable/views/__init__.py]: Thought about it but then assumed the wrapping already makes it clear

Comment (pferreir) [inline indico/modules/events/timetable/views/__init__.py]: The wrapping per se doesn't make it clear, IMHO, but anyway, not very important.

Comment (ThiefMaster) [inline indico/modules/events/timetable/views/__init__.py]: updated

**Pull request merged**